internal/api/oidc/client.go

```diff
@@ -125,11 +124,11 @@ func (o *OPStorage) SetUserinfoFromToken(ctx context.Context, userInfo oidc.User
 		return errors.ThrowPermissionDenied(nil, "OIDC-Dsfb2", "token is not valid or has expired")
 	}
 	if token.ApplicationID != "" {
-		app, err := o.repo.ApplicationByClientID(ctx, token.ApplicationID)
+		app, err := o.query.AppByID(ctx, token.ApplicationID)
```

Comment:
```suggestion
		app, err := o.query.AppByOIDCClientID(ctx, token.ApplicationID)
```

Reply:
in token view there's a naming / using error where the client_id is stored in `ApplicationID`

we'll have to fix this with the new token projection:
either rename it or change the "content"

Reply:
do we have a tracking issue for that? I received a 418 from https://livios.brain/caos/zitadel/issues?name=auth%20request%20refactoring  🥴

(added a comment on the tracking issue of queries)